smithy-smoke-test-traits/src/main/resources/META-INF/smithy/smithy.test.smithy

```diff
@@ -0,0 +1,91 @@
+$version: "2.0"
+
+namespace smithy.test
+
+/// Defines a set of test cases to send to a live service to ensure that a
+/// client can successfully connect to a service and get the right kind of
+/// response.
+@trait(selector: "operation")
+list smokeTests {
+    /// A list of smoke tests to perform on the operation
+    member: SmokeTestCase
+}
+
+/// A single smoke test case definition.
+@private
```

Comment:
What is and isn't private isn't consistent

Reply:
Made `Tag` private. Is that all you meant?